Tests/NetworkingTests/ErrorProcessorTests.swift

```diff
@@ -113,7 +113,7 @@ final class ErrorProcessorTests: XCTestCase {
 extension ErrorProcessorTests {
     func test_apiManager_request_errorShouldBeMappedToSimpleError() async {
         let apiManager = APIManager(
-            urlSession: URLSession.shared,
+            responseProvider: URLSession.shared,
```

Comment:
As mentioned in `APIManager` I would rather see the `urlSession` parameter name here